WindowsServerDocs/identity/ad-fs/operations/Configure-AD-FS-Extranet-Smart-Lockout-Protection.md

```diff
@@ -280,7 +280,7 @@ A: If the clients connect directly to the ADFS servers and not via Web Applicati
 A: ESL will work well to prevent Exchange Online or other legacy authentication brute force attack scenarios. A legacy authentication has an “Activity ID” of 00000000-0000-0000-0000-000000000000. In these attacks, the bad actor is taking advantage of Exchange Online basic authentication (also known as legacy authentication) so that the client IP address appears as a Microsoft one. The Exchange online servers in the cloud proxy the authentication verification on behalf of the Outlook client. In these scenarios, the IP address of the malicious submitter will be in the x-ms-forwarded-client-ip and the Microsoft Exchange Online server IP will be in the x-ms-client-ip value.
 Extranet Smart Lockout checks network IPs, forwarded IPs, the x-forwarded-client-IP, and the x-ms-client-ip value. If the request is successful, all the IPs are added to the familiar list. If a request comes in and any of the presented IPs are not in the familiar list then the request will be marked as unfamiliar. The familiar user will be able to sign in successfully while requests from the unfamiliar locations will be blocked.  
 
-**Q: Can I estimate the size of the ADFSArtifactStore before enabling ESL?
+**Can I estimate the size of the ADFSArtifactStore before enabling ESL?
```

Comment:
in addition, please also add the missing bold end formatting (**):
```suggestion
**Can I estimate the size of the ADFSArtifactStore before enabling ESL?**
```
(Took me a few moments to realize that this FAQ section has moved on from the old-school "Q: -- A:" format to now use **bold** for the questions instead.)